src/CommerceModel.BetterRetail/artifacts/OOE/BetterRetail/QueueOrderSchemaImportActivity/providers.json

```diff
@@ -1427,13 +1358,13 @@
       "MapperPackageType": "Boîte Std.|Std. 31 kg pkg.|Std. Box=BOX",
       "MapperWeightUOM": "Pound=LB,Ounce=OZ,Kilogram=KG",
       "MapperQuantityUOM": "",
-      "CarrierMappingConfiguration": "{'Express':{'SAvER':{'UpS': 'UpS','DhL': 'DhL'},'RapIdo':{'MexIco':'MexIco','ChIli':'ChIli'}}, 'TimesAver':{'SaVer':{'UpS':'UpS','dHl':'dHl'}, 'Saving':{'UpS': 'UpS', 'FeDEx' : 'FeDEx'}}, 'StanDarD':{'SAver':{'uPS': 'uPS', 'Dhl': 'Dhl'}}}",
+      "CarrierMappingConfiguration": #TranssmartMappingNL#,
```

Comment:
There needs to be quotes around the mapping settings....

Reply:
Those values are persisted using XML format in the database and I am not sure our provider mechanism supports complex objects. Please, test it on your env first, deployment and package booking . Anyway, I am still not sure whether it should be parametrized.

Reply:
Oh, and "CarrierMappingConfiguration" is defined as a string in the provider class definition.

Reply:
Changes to CarrierMappingConfiguration were reverted in favor of a new JSON that better represents actual testing data for that provider.